OxfordBodleian/Juel-Jensen/BDLaethe32.xml

```diff
@@ -189,21 +636,222 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <desc type="GuestText"><title type="complete" ref="LIT4145Onausp"/></desc>
+                                    <q xml:lang="gez"><seg part="I"><gap reason="illegible"/><sic>መስከም፡</sic>
+                                        <hi rend="rubric" rendition="#partialRubric">፲</hi>ወ<hi rend="rubric" rendition="#partialRubric"><gap reason="illegible"/></hi>
+                                        ፅንዕ<gap reason="illegible"/> ወእምኔሁ፡ ጽንጽንት። በ<gap reason="illegible"/>ቅምት፡ 
+                                        <hi rend="rubric" rendition="#partialRubric">፳</hi>ወ<hi rend="rubric" rendition="#partialRubric">፰</hi> ጽንጽንት፡
+                                    ወእምኔሁ፡ ጽንዕት።</seg></q>
+                                    <q xml:lang="gez"><seg part="F">በነሐሴ፡ <hi rend="rubric" rendition="#partialRubric">፳</hi>ወ<hi rend="rubric" rendition="#partialRubric">፫</hi>
+                                        ጽንዕት፡ ወእምኔሁ<supplied reason="omitted">፡</supplied> ጽንጽንት፡ በጳጉሜ፡ <hi rend="rubric" rendition="#partialRubric">፫</hi> ጽንጽንት፡ ወእምኔሁ፡ ጽንዕት<supplied reason="omitted">፡</supplied></seg></q>
+                                </item>
+                                
+                                <item xml:id="a17">
+                                    <locus target="#121vb"/>
+                                    <desc type="GuestText"><title type="incomplete" corresp="NAR0005calend"/> Note listing the feast days of the apostles for each month. The text breaks off at the end.</desc>
+                                    <q xml:lang="gez"><seg part="I">ሠረቀ፡ መስከረም፡ በ<hi rend="rubric" rendition="#partialRubric">፩</hi> በርተሎሜዎስ፡ ዮሐንስ፡ ምጥምቅ፡ ሠረቀ፡ ጥቅምት፡ 
+                                        በ<hi rend="rubric" rendition="#partialRubric">፲</hi>ወ<hi rend="rubric" rendition="#partialRubric">፪</hi>፡
+                                        ማቴዎስ፡ ወንጌላዊ<hi rend="rubric" rendition="#partialRubric">፨</hi></seg></q>
+                                    <q xml:lang="gez"><seg part="F">ሠረቀ፡ ሐምሌ፡ በ<hi rend="rubric" rendition="#partialRubric">፪</hi> ታዴዎስ፡
+                                        በ<hi rend="rubric" rendition="#partialRubric">፭</hi> ጴጥሮስ፡ ወጳውሎስ፡ በ<hi rend="rubric" rendition="#partialRubric">፲</hi>
+                                        ናትናኤል፡ በ<hi rend="rubric" rendition="#partialRubric">፲</hi>ወ<hi rend="rubric" rendition="#partialRubric">፰</hi>
+                                        ያዕቆብ፡ እሁሁ፡ <sic>ለእእግዚእነ፡</sic> ሠረቀ፡ ነሐሴ፡ በ<hi rend="rubric" rendition="#partialRubric">፲</hi>ወ<hi rend="rubric" rendition="#partialRubric">፭</hi>
+                                    ጉቡአን፡</seg></q>
+                                </item>
+                                
+                                <item xml:id="a18">
+                                    <locus target="#122r"/>
+                                    <desc type="GuestText">Partly effaced and hardly legible note in red. Possibly a calendaric note in Amharic.</desc>
```

Comment:
Concerning the pink note on the bottom:

To me it seems that the first two lines are the beginning of a calendaric text (the same as on the opposite side). The following lines seem to be letters according to the order of the "alphabet":

ፀ ፁ ፂ ፃ ፄ ፅ ፆ
ፋ ፉ ፊ ፋ ፌ ፍ ፎ
ፐ ፑ ፒ ፓ ፔ ፕ ፖ
ኰ . . .
ኈ . . .
ቈ . . .